test/call-test.js

```diff
@@ -380,4 +395,36 @@ describe("Call API", function () {
 			return client.Call.sendDtmf(testCall.id, dtmfOut.dtmfOut);
 		});
 	});
+
+	describe("pagination tests", function () {
+
+		before(function () {
+			nock("https://api.catapult.inetwork.com")
+			.persist()
+			.get("/v1/users/" + userId + "/calls")
+			.reply(200, callsList,
+				{
+					"link" : "<https://api.catapult.inetwork.com/v1/users/" + userId +
+					"/calls?page=0&size=25>; rel=\"first\"," +
+					"<https://api.catapult.inetwork.com/v1/users/" + userId + "/calls>; rel=\"next\""
+				});
+			nock.disableNetConnect();
+		});
+
+		after(function () {
+			nock.cleanAll();
+			nock.enableNetConnect();
+		});
+
+		it("should return a list of calls with a page to the next link", function () {
+			return client.Call.list({})
+			.then(function (callResponse) {
+				callResponse.calls.should.eql(callsList);
+				return callResponse.getNextPage();
+			})
+			.then(function (moreCalls) {
+				moreCalls.calls.should.eql(callsList);
+			});
+		});
+	});
 });
```

Comment:
Would it be better to split this into two `it` statements: 
```
it("should return a list of calls", function () {
 ...
});
it("with a link to the next page", function () {
...
});
```
so that we can distinguish between those different kinds of test failures?